server/forge/forge.go

```diff
@@ -32,6 +32,9 @@ type Forge interface {
 	// Name returns the string name of this driver
 	Name() string
 
+	// Link returns the root url of a configured forge
+	Link() string
```

Comment:
Shouldn't we call it url then?

Reply:
no that is explizite link ...

https://woodpecker-ci.org/docs/next/usage/environment#built-in-environment-variables

Do use always use `_LINK` only one time we have URL ... and I'm for consistency ...

Reply:
-> #1793

Reply:
ok let's do a vote: https://framadate.org/mOJABOjBGDEVPsyv